src/test/java/net/snowflake/client/jdbc/ConnectionIT.java

```diff
@@ -543,7 +547,12 @@ public void testDifferentKeyLength() throws Exception {
       properties.put("privateKey", privateKey);
       try (Connection connection = DriverManager.getConnection(uri, properties);
           Statement statement = connection.createStatement()) {
-        statement.execute(String.format("alter user %s unset rsa_public_key", testUser));
+        assertFalse(connection.isClosed());
+      } finally {
+        try (Connection connection = getConnection();
+            Statement statement = connection.createStatement()) {
+          statement.execute(String.format("alter user %s unset rsa_public_key", testUser));
```

Comment:
Now this test fails with `SQL access control error:
Insufficient privileges to operate on user 'SNOWMAN'`. Somehow `accountadmin` role from properties is not propagated to the connection(?) and adding a `statement.execute("use role accountadmin");` before this line.

Reply:
You are right, done.